packages/itwinui-css/src/header/header-buttons.scss

```diff
@@ -160,6 +160,17 @@ $iui-side-navigation-icon-margins: calc(1.5 * var(--iui-size-m));
   &[aria-disabled='true'] {
     cursor: not-allowed;
   }
+
+  svg {
```

Comment:
Note: This is fine for now because we use this pattern in many places, but we should start thinking about how to consolidate the styling for icons to reduce duplication and make it easier to adjust.

We shouldn't target loose `svg` selectors like this; instead we should be try modify a custom property (e.g. `--_iui-svg-fill`) in the parent itself.